src/uct/base/uct_iface.c

```diff
@@ -487,7 +487,7 @@ UCS_CLASS_INIT_FUNC(uct_base_iface_t, uct_iface_ops_t *ops, uct_md_h md,
         alloc_methods_bitmap |= UCS_BIT(method);
     }
 
-    self->config.failure_level = config->failure;
+    self->config.failure_level = (ucs_log_level_t)config->failure;
```

Comment:
unrelated fix, i would put in another PR (i'm sure we could have more type castings to add)